source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -200,7 +200,6 @@ - (void)source:(ACVTableViewController *)avcTabVc userconfig:(NSString *)payload
 {
     self.hostconfig = payload;
 }
-
 - (void)didFetchUserResponses:(NSData *)json error:(NSError *)error
```

Comment:
revert

Reply:
i don't understand this comment

---
In reply to: [155119854](https://github.com/Microsoft/AdaptiveCards/pull/1029#discussion_r155119854) [](ancestors = 155119854)

Reply:
You are removing a line here. Why?

---
In reply to: [155120091](https://github.com/Microsoft/AdaptiveCards/pull/1029#discussion_r155120091) [](ancestors = 155120091,155119854)